case-lib/lib.sh

```diff
@@ -327,13 +327,6 @@ die()
     exit 1
 }
 
-# force ask buffer data write into file system
-sudo sync -f || true
-# catch kern.log last line as current case start line
-if [ ! "$DMESG_LOG_START_LINE" ]; then
-    DMESG_LOG_START_LINE=$(wc -l /var/log/kern.log|awk '{print $1;}')
-fi
-
 is_sof_used()
```

Comment:
In an ideal world every commit should be usable for bisecatability, cherry-picking, reverting, etc. reasons. I guess this implies the entire `DMESG_LOG_START_LINE` removal should be a single commit?